upgrade/UPGRADE-v9.0.0-dev.md

```diff
@@ -1374,6 +1374,9 @@ There you can find links to upgrade notes for other versions too.
 - remove unnecessary entity extensions from parameters_common.yml ([#1663](https://github.com/shopsys/shopsys/pull/1663))
     - there is no longer need to register entities in `App` namespace extending entities from `Shopsys` namespace, so remove all necessary uses, see #project-base-diff for example
 
+- update to strict comparison in order to avoid errors ([#1782](https://github.com/shopsys/shopsys/pull/1782))
```

Comment:
What about?
```suggestion
- use strict comparison in category panel template to prevent errors ([#1782](https://github.com/shopsys/shopsys/pull/1782))
```